website/src/app/[lang]/[region]/(website)/transparency/finances/[currency]/section-1.tsx

```
@@ -3,7 +3,7 @@ import { Card, CardContent, Typography } from '@socialincome/ui';
 import _ from 'lodash';
 import { SectionProps } from './page';
 
-export const roundAmount = (amount: number) => Math.round(amount / 10) * 10;
+export const roundAmount = (amount: number) => (!isNaN(amount) ? Math.round(amount / 10) * 10 : 0);
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)